build.js

```diff
@@ -15,4 +15,7 @@ const minified = MINIFY && terser.minify(jsSourceProcessed, {
   }
 });
 
+const nodeHeader = `import { TextEncoder } from 'util';`;
```

Comment:
Did you mean to use `require` here?

Reply:
This is before it's run through babel. If at some point you decide to ship es module in node, this would still work.